### ISSUE TITLE
Redis auth config? Screenshots?

### ISSUE THREAD

**Issue by GeoffreyPlitt** [OptimalBits/bull-ui#1]
Any documentation of how to send redis auth?

Any screenshots?

**Comment by manast** [comment]
no, this is just a barebones project. Can be built upon it, but I do not have time right now...
